### PR TITLE
`ComputeSlopeV2` + `BigIntToUint256 ` hints

### LINKED ISSUE #531: `ComputeSlopeV2` hint  (secp hint)
whitelisted in vrf.json : 

https://github.com/starkware-libs/cairo-lang/blob/0e4dab8a6065d80d1c726394f5d9d23cb451706a/src/starkware/starknet/security/whitelists/vrf.json#L279

https://github.com/lambdaclass/cairo-vm/blob/db7fff18c9c1312024ebf4119fcdfeba23657b12/vm/src/hint_processor/builtin_hint_processor/hint_code.rs#L724

### LINKED ISSUE #554: `BigIntToUint256` hint
https://github.com/lambdaclass/cairo-vm/blob/db7fff18c9c1312024ebf4119fcdfeba23657b12/vm/src/hint_processor/builtin_hint_processor/hint_code.rs#L307

https://github.com/starkware-libs/cairo-lang/blob/0e4dab8a6065d80d1c726394f5d9d23cb451706a/src/starkware/cairo/common/cairo_secp/bigint.cairo#L82

https://github.com/starkware-libs/cairo-lang/blob/0e4dab8a6065d80d1c726394f5d9d23cb451706a/src/starkware/starknet/security/whitelists/0.10.3.json#L660

### PR DESCRIPTION
**Pull request by TAdev0**
Resolves #531 
Resolves #554

### CODE REVIEW

**Comment by TAdev0** [comment]
@cicr99  @har777  ready for review !
All 3 hints work properly, the integration test involves the usage of the 3 , it works as expected

File: pkg/hintrunner/zero/zerohint_ec.go [hunk 1]
```diff
@@ -11,11 +11,83 @@ import (
...
+			xBigInt, err := vm.Memory.ResolveAsBigInt3(xAddr)
+			if err != nil {
+				return err
+			}
+
+			var xD0Big big.Int
+			(*xBigInt[0]).BigInt(&xD0Big)
+
+			var xD1Big big.Int
+			(*xBigInt[1]).BigInt(&xD0Big)
```
Comment (har777) [inline pkg/hintrunner/zero/zerohint_ec.go:50]: Did you mean `(*xBigInt[1]).BigInt(&xD1Big)` here

Comment (TAdev0) [reply pkg/hintrunner/zero/zerohint_ec.go:50]: oh yeah... how is it possible it works like this? aha

Comment (TAdev0) [reply pkg/hintrunner/zero/zerohint_ec.go:50]: ok i see , `BigIntToUint256` was imported in the cairo file but not used ><

**Comment by TAdev0** [comment]
@har777  i added a test for the hint that was wrong and was not tested. I get some SIGSEGV error for this file i simlified as much as i could:

```
func main{range_check_ptr: felt}() {
    // Bigint to uint
    let big_int = BigInt3(d0=9, d1=9, d2=9);

    let (uint256) = bigint_to_uint256(big_int);

    return ();
}

```

```
     Running....
        panic: runtime error: invalid memory address or nil pointer dereference
        [signal SIGSEGV: segmentation violation code=0x2 addr=0x8 pc=0x104ec9550]
        ```

**Comment by TAdev0** [comment]
@cicr99  @har777  @MaksymMalicki  SIGSEGV error resolved, ready for final review

File: pkg/hintrunner/zero/zerohint_ec.go [hunk 2]
```diff
@@ -768,6 +839,125 @@ func createComputeSlopeV1Hinter(resolver hintReferenceResolver) (hinter.Hinter,
...
+				return err
+			}
+
+			//> y0 = pack(ids.point0.y, PRIME)
+			y0Big, err := secp_utils.SecPPacked(point0YValues)
+			if err != nil {
+				return err
+			}
+
+			//> y1 = pack(ids.point0.y, PRIME)
```
Comment (har777) [inline pkg/hintrunner/zero/zerohint_ec.go:922]: this comment is wrong?

**Review by har777 — APPROVED** [review]
Just one mistake in a comment in the code.

Also I see the PR title mentions `VerifyZeroV3` which seems missing in this PR?

**Review by har777 — COMMENTED** [review]
Just notices that unit tests are missing 😅

**Review by har777 — CHANGES_REQUESTED** [review]
Just notices that unit tests are missing 😅

**Comment by TAdev0** [comment]
@har777  just added unit tests for BigIntToUint256. will wait for  #549  being merged to refactor ComputeSlopeV2

File: pkg/hintrunner/zero/zerohint_ec.go (hunk 2 above)
Comment (TAdev0) [reply pkg/hintrunner/zero/zerohint_ec.go:1101]: why? i checked again and it looks fine?

**Comment by TAdev0** [comment]
@har777  VerifyZeroV3 seems missing because i implemented it in another PR as well (because i needed it there) so it just disappeared in the diff. #552  is already closed

**Comment by TAdev0** [comment]
@har777  @cicr99  ready for review. There is no unit test for `ComputeSlopeV2` because like ComputeSlopeV3, computation is already checked in `ComputeSlope` test case that uses `ComputeSlopeV1` hint

File: pkg/hintrunner/zero/zerohint_ec.go (hunk 2 above)
Comment (har777) [reply pkg/hintrunner/zero/zerohint_ec.go:1251]: the initial code comment and your code indicate `//> y1 = pack(ids.point1.y, PRIME)` not `//> y1 = pack(ids.point0.y, PRIME)`?

Comment (TAdev0) [reply pkg/hintrunner/zero/zerohint_ec.go:1251]: oooh yes thank you Hari!

**Review by har777 — APPROVED** [review]

**Pull request merged**